app/src/main/res/layout/viewflipper_setting_ip.xml

```diff
@@ -47,6 +65,17 @@
                 android:scrollHorizontally="true"
                 android:text="http://www.xwiki.org/xwiki"
                 android:textColor="@android:color/black" />
+            <Button
+                android:layout_weight="1"
+                android:layout_width="100dp"
```

Comment:
Follow the android material design guidelines for sizes and styles, do not add any random values, refer to guidelines first: https://material.io/guidelines/components/buttons.html#buttons-style